code/modules/mob/dead/new_player/sprite_accessory/tails.dm

```diff
@@ -341,3 +341,10 @@
 	icon_state = "hearttail"
 	color_key_defaults = list(KEY_SKIN_COLOR)
 	can_wag = TRUE
+
+/datum/sprite_acessory/tail/rattlesnake
```

Comment:
```suggestion
/datum/sprite_accessory/tail/rattlesnake
```

simple typo, should compile when you fix it. make sure to get a screenshot of how these look in game.